rightrotate.py

```diff
@@ -0,0 +1,24 @@
+# Python program to right rotate a list by n 
+
+
+def rightRotate(lists, num): 
+    output_list = [] 
+      
+    # Will add values from n to the new list 
+    for item in range(len(lists) - num, len(lists)): 
+        output_list.append(lists[item]) 
+      
+    # Will add the values before 
+    # n to the end of new list     
+    for item in range(0, len(lists) - num):  
+        output_list.append(lists[item]) 
+          
+    return output_list 
+  
+# Driver Code 
+
+k = 4
+n = [7, 8, 9, 10, 11, 12] 
+  
```

Comment:
hello @nancyanand2807 try to remove the unnecessary lines after each statement and do generalized code for this problem. your code is perfect for the given list and the given k.
make it a generic one such that it runs for every possible value of k and the user-defined list.

Reply:
Sure. I will update it in some time.

Reply:
ok

Reply:
Hey, I have updated the code as per the requirements. Kindly review it.